py/selenium/webdriver/support/expected_conditions.py

```diff
@@ -95,6 +95,8 @@ def __call__(self, ignored):
 
 
 def _element_if_visible(element, visibility=True):
+    if isinstance(element, str) or isinstance(element, dict):
+           raise StaleElementReferenceException("Invalid locator")
```

Comment:
<selenium.webdriver.remote.webelement.WebElement (session="49274937103ea6cd11bc1c9a3a4784ee", element="0.9158218795761794-1")>
<selenium.webdriver.remote.webelement.WebElement (session="49274937103ea6cd11bc1c9a3a4784ee", element="0.9158218795761794-2")>
Traceback (most recent call last):
  File "D:\Dropbox\Dropbox\Meu Curso de Selenium\testes\loading_page.py", line 30, in <module>
    test = wait_element_not_present(driver, ".loadingoverlay")
  File "D:\Dropbox\Dropbox\Meu Curso de Selenium\testes\loading_page.py", line 20, in wait_element_not_present
    expected_conditions.invisibility_of_element_located((By.CSS_SELECTOR, elem))
  File "C:\Python35-32\lib\site-packages\selenium\webdriver\support\wait.py", line 71, in until
    value = method(self._driver)
  File "C:\Python35-32\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 198, in __call__
    return _element_if_visible(_find_element(driver, self.locator), False)
  File "C:\Python35-32\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 100, in _element_if_visible
    return element if element.is_displayed() == visibility else False
AttributeError: 'dict' object has no attribute 'is_displayed'